tmt/steps/execute/internal.py

```diff
@@ -32,7 +32,7 @@
 TEST_PIDFILE_LOCK_FILENAME = f'{TEST_PIDFILE_FILENAME}.lock'
 
 #: The default directory for storing test pid file.
-TEST_PIDFILE_ROOT = Path('/var/tmp')
+TEST_PIDFILE_ROOT = '/var/tmp'  # noqa: S108
```

Comment:
What is wrong with making it Path here?

Reply:
Nothing wrong. It just made more sense to me when I was doing the changes. Can revert.

Reply:
Please, do so, it's not just any string, it's a *path* :)

Reply:
Will do.
And it needs to be /var/tmp, yes?
I assume the whole 'needs to survive reboot' is not super importand for pidfile dir. 
`mkdtemp` with a prefix perhaps? :)

Reply:
Speaking of which, shouldn't it be user-specific. What if this is run in an hpc with multiple simultaneous users? (Would have more issues with installation and stuff, but still)

Reply:
It must be possible to find this directory not just by tmt, but also by tools that are running outside of the context of a test when an out-of-session reboot is needed. `tmt-reboot` executed by a user in their SSH session must be able to find a directory that tmt used to store pid of the test it's running in completely different SSH session. Therefore it seems to be the predictability is necessary - I suppose could get away with something like `/var/run` and so on, there might be (solvable) permission issues, but it needs to be predictable, i.e. I don't see `mkdtemp` working reliably.

Reply:
It may become user-specific, but then again, let's say guests A, B and C are running their tests on a guest, and user A runs `tmt-reboot` in another of their SSH session. Per-user pid file would be fine, it'd nicely notify A's tmt to stop the test and reboot the guest - but B's and C's tmt runs would be left in the dark. I suppose this fits the shelf of issues related to teaching tmt to share a guest with other unrelated tmts, as there are no strong guarantees when it comes to sharing...

Reply:
Path is back; Keeping /var/tmp/.